CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+Wrangler is an open source project because we believe that folks should have access, insight,
+and the opportunity to contirbute to their developer tools. Wrangler is also a product
+delivered by Cloudflare, so it's important to clarify how we think about issue triage and
+contributions.
+
+## People
+
+Wrangler is maintained by @ashleygwilliams, and her team, [Workers Developer Experience].
+
+[Workers Developer Experience]: https://github.com/orgs/cloudflare/teams/workers-devexp
+
+## Primary Issue Triage
+
+Within 3 days, any incoming issue should be triaged. Triage involves:
+
+- reading the issue and requesting any further information
+- always thank the user for submitting
+
+### Labelling
+
+- label all issues coming from non-team members with User Report
+- labelling the category of the issue: Feature, External Bug, Bug, Maintenance, Docs
+- optionally labelling a secondary category: Webpack, Routes, Workers Runtime, Refactor
+- labelling the status of the issue: Need More Info, Needs Repro, Needs Design, PR Welcome
+- optionally labelling other calls to action: Help Wanted, Question
+
+### Assignment
+
+- if the issue will require a large amount of back and forth between the reporter and the team
+    assign a single team member to manage the conversation
+
+## Product Issue Triage
+
+Once a week, the team meets to do Product Triage. This is where we assign work and update
+our plans for the milestones and releases.
+
+### Labelling
+
+- labelling the priority of the issue: Critical, Nice to Have, Low Priority
```

Comment:
I think this should be done in a meeting, priorities can be relative; we should all agree.

Reply:
this is done in a meeting- it says in the prose under the title ;)